dataeng/jobs/analytics/LoadEvents.groovy

```diff
@@ -0,0 +1,142 @@
+                stringParam('SCHEMA', allVars.get('SCHEMA'))
+            }
+            multiscm common_multiscm(allVars)
+            wrappers common_wrappers(allVars)
+            publishers common_publishers(allVars)
+            steps {
+                shell(dslFactory.readFileFromWorkspace('dataeng/resources/load-events-to-vertica.sh'))
+            }
+        }
+    }
+
+    public static def load_json_events_to_s3_job = { dslFactory, allVars ->
+        dslFactory.job("load-json-events-to-s3") {
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters from_date_interval_parameter(allVars)
+            parameters to_date_interval_parameter(allVars)
+            parameters {
+                stringParam('OUTPUT_URL', allVars.get('OUTPUT_URL'), '')
+                stringParam('CREDENTIALS', allVars.get('CREDENTIALS'), '')
+                stringParam('EVENTS_LIST', allVars.get('EVENTS_LIST'), '')
+                stringParam('EXTRA_ARGS', allVars.get('EXTRA_ARGS', '--event-record-type JsonEventRecord'), '')
+            }
+            multiscm common_multiscm(allVars)
+            triggers common_triggers(allVars)
+            wrappers common_wrappers(allVars)
+            publishers common_publishers(allVars)
+            publishers {
+                downstreamParameterized {
+                    trigger('load-json-events-to-bigquery') {
+                        condition('SUCCESS')
+                        parameters {
+                            predefinedProp('CLUSTER_NAME', 'LoadJsonEventsToBigQueryAutomatic')
+                            predefinedProp('OUTPUT_URL', '$OUTPUT_URL')
+                            predefinedProp('FROM_DATE', '$FROM_DATE')
+                            predefinedProp('TO_DATE', '$TO_DATE')
+                        }
+                    }
+                }
+            }
+            steps {
+                shell(dslFactory.readFileFromWorkspace('dataeng/resources/load-events-to-s3.sh'))
+            }
+        }
+    }
+
+    public static def load_json_events_to_bigquery_job = { dslFactory, allVars ->
+        dslFactory.job("load-json-events-to-bigquery") {
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters from_date_interval_parameter(allVars)
+            parameters to_date_interval_parameter(allVars)
+            parameters {
+                stringParam('OUTPUT_URL', allVars.get('OUTPUT_URL', ''))
+                stringParam('CREDENTIALS', allVars.get('CREDENTIALS', ''))
+                stringParam('EXTRA_ARGS', allVars.get('EXTRA_ARGS', ''),
+                '' + $/
+  Examples:
+
+  * --event-record-type JsonEventRecord
+  * --max-bad-records=1
+  * --overwrite
+  /$)
+                stringParam('DATASET', allVars.get('DATASET', ''))
+            }
+            multiscm common_multiscm(allVars)
+            triggers common_triggers(allVars)
```

Comment:
does this still need common_triggers if it's being triggered by upstream?

Reply:
Good catch. Removed